test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -729,6 +729,190 @@ test.describe( 'Image', () => {
 	} );
 } );
 
+test.describe( 'Image - interactivity', () => {
+	let postId = null;
+	let filename = null;
+
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllMedia();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllMedia();
+	} );
+
+	test.beforeEach( async ( { admin, page, editor, imageBlockUtils } ) => {
+		await admin.visitAdminPage(
+			'/admin.php',
+			'page=gutenberg-experiments'
+		);
+
+		await page
+			.locator( `#gutenberg-interactivity-api-core-blocks` )
+			.setChecked( true );
+
+		await page.locator( `input[name="submit"]` ).click();
+		await page.waitForLoadState();
+
+		await admin.createNewPost();
+		await editor.insertBlock( { name: 'core/image' } );
+
+		const imageBlock = page.locator(
+			'role=document[name="Block: Image"i]'
+		);
+		await expect( imageBlock ).toBeVisible();
+
+		filename = await imageBlockUtils.upload(
+			imageBlock.locator( 'data-testid=form-file-upload-input' )
+		);
+		const image = imageBlock.locator( 'role=img' );
+		await expect( image ).toBeVisible();
+		await expect( image ).toHaveAttribute( 'src', new RegExp( filename ) );
+
+		await editor.openDocumentSettingsSidebar();
+	} );
+
+	test.afterEach( async ( { requestUtils, admin, page } ) => {
+		await requestUtils.deleteAllMedia();
+
+		await admin.visitAdminPage(
+			'/admin.php',
+			'page=gutenberg-experiments'
+		);
+
+		await page
+			.locator( `#gutenberg-interactivity-api-core-blocks` )
+			.setChecked( false );
+
+		await page.locator( `input[name="submit"]` ).click();
+
+		await page.waitForLoadState();
+	} );
+
+	test( 'should toggle "lightbox" in saved attributes', async ( {
+		editor,
+		page,
+	} ) => {
+		await page.getByRole( 'button', { name: 'Advanced' } ).click();
+		await page.getByLabel( 'Behaviors' ).selectOption( 'Lightbox' );
```

Comment:
I've tried using `getByRole` here without success. I've tried various different forms of the following syntax, but it doesn't find the select:

```
const select = page.getByRole( 'select' ).filter( {
	has: page.getByRole( 'option', { value: 'lightbox' } ),
} );
await select.selectOption( 'lightbox' );
```

Reply:
The role is likely `combobox` rather than `select`.

Try something like:

```js
page.getByRole( 'region', { name: 'Editor settings' } )
  .getByRole( 'combobox', { name: 'Behaviors' } )
```

Reply:
Perfect, thanks! Searching for `combobox` worked.